acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/signing/SigningAcceptanceTestBase.java

```diff
@@ -37,7 +37,18 @@ protected void setupSigner(final String mode, final Map<String, String> env) {
   }
 
   protected Bytes verifyAndGetSignatureResponse(final Response response) {
-    response.then().contentType(ContentType.TEXT).statusCode(200);
-    return Bytes.fromHexString(response.body().print());
+    return verifyAndGetSignatureResponse(response, ContentType.TEXT);
+  }
+
+  protected Bytes verifyAndGetSignatureResponse(
+      final Response response, final ContentType contentType) {
+    response.then().contentType(contentType).statusCode(200);
+    final String signature;
+    if (contentType == ContentType.JSON) {
```

Comment:
Doesn't look like this is actually use for the json content type. Would be good to have an AT that shows we return json. The ableToSignWithoutSigningRootField only checks the contentType is set correctly, it doesn't check the response.